app/src/ui/about/about.tsx

```diff
@@ -269,6 +273,18 @@ export class About extends React.Component<IAboutProps, IAboutState> {
       return null
     }
 
+    if (isWindowsAndNoLongerSupportedByElectron()) {
+      return (
+        <DialogError>
+          This operating system is no longer supported. Software updates have
+          been disabled.{' '}
```

Comment:
Not a stopper for this PR, of course, but now that we're doing a lot of accessibility work I noticed this text/component probably doesn't meet the accessibility contrast requirements. I dealt with that (but in another component) here: https://github.com/desktop/desktop/pull/16567/files#diff-c6834002e3d30343d10d7dc537a606eb8d55e36d1ce5d3d41419123f3e8f2343

Reply:
Hmm.. I think it does?

<img width="592" alt="image" src="https://user-images.githubusercontent.com/634063/234025271-c15d0e5b-192b-44d4-b071-362aebfe1bbd.png">